api/src/test/java/org/openmrs/util/databasechange/ValidateHibernateMappingsDatabaseIT.java

```diff
@@ -81,9 +82,8 @@ private SessionFactory buildSessionFactory() {
 		Configuration configuration = new Configuration().configure();
 		
 		Set<Class<?>> entityClasses = OpenmrsClassScanner.getInstance().getClassesWithAnnotation(Entity.class);
-		if (entityClasses.contains(OrderServiceTest.SomeTestOrder.class)) {
```

Comment:
Why did we remove the above check?

Reply:
We don't need to check if an entity exists in an ArrayList before removing it: https://docs.oracle.com/javase/8/docs/api/java/util/ArrayList.html#remove-java.lang.Object-